mongo/cursor_test.go

```diff
@@ -157,5 +157,15 @@ func TestCursor(t *testing.T) {
 			assert.Nil(t, err, "All error: %v", err)
 			assert.True(t, tbc.closed, "expected batch cursor to be closed but was not")
 		})
+
+		t.Run("does not error given interface as parameter", func(t *testing.T) {
```

Comment:
The test could benefit from some whitespace for readability. See the test above for an example grouping.

Reply:
done!